packages/components/src/components/form/Select/Select.tsx

```diff
@@ -1,243 +1,173 @@
+            }
+        }
+    }
+
+    .${reactSelectClassNamePrefix}__placeholder {
+        color: ${({ theme, isDisabled }) => (isDisabled ? theme.textDisabled : theme.textSubdued)};
+        ${typography.body}
+    }
+
+    .${reactSelectClassNamePrefix}__value-container {
+        display: flex;
+        flex-wrap: nowrap;
+        min-width: ${({ minValueWidth }) => minValueWidth};
+        justify-content: ${({ isClean }) => (isClean ? 'flex-end' : 'flex-start')};
+        padding: 0;
+        border: none;
+    }
+
+    .${reactSelectClassNamePrefix}__single-value {
+        position: static;
+        display: flex;
+        align-items: center;
+        justify-content: ${({ isClean }) => (isClean ? 'flex-end' : 'flex-start')};
+        width: 100%;
+        max-width: initial;
+        color: ${({ isDisabled, theme }) => (isDisabled ? theme.textDisabled : theme.textDefault)};
+        border-style: none;
+        transform: none;
+
+        :hover {
+            cursor: ${({ isSearchable }) => isSearchable && 'text'};
+        }
+    }
+
+    .${reactSelectClassNamePrefix}__input {
+        color: ${({ theme }) => theme.textDefault} !important;
+
+        ${typography.body};
+    }
+
+    .${reactSelectClassNamePrefix}__indicator-separator {
+        display: none;
+    }
+
+    .${reactSelectClassNamePrefix}__menu {
+        ${menuStyle}
+        border: none;
+        z-index: ${zIndices.base};
+    }
+
+    .${reactSelectClassNamePrefix}__group-heading {
+        margin: 0;
+        padding: ${spacingsPx.xs};
+        ${typography.label};
+        text-transform: initial;
+    }
+
+    .${reactSelectClassNamePrefix}__group {
+        padding: 0;
+
+        & + & {
+            padding-top: ${spacingsPx.xxs};
+            margin-top: ${spacingsPx.xxs};
+        }
+    }
+
+    .${reactSelectClassNamePrefix}__option {
+        padding: ${spacingsPx.xs} ${spacingsPx.sm};
+        border-radius: ${borders.radii.xs};
+        color: ${({ theme }) => theme.textDefault};
+        ${typography.body};
+        cursor: pointer;
+
+        :active {
+            background: ${({ theme }) => theme.backgroundSurfaceElevation0};
+        }
+    }
 `;
 
 // Prevent closing the menu when scrolling through options.
 const closeMenuOnScroll = (e: Event) =>
     !(e.target as Element)?.className?.startsWith(reactSelectClassNamePrefix);
 
-type Option = any;
-
-/** Custom Type Guards to check if options are grouped or not */
-const isOptionGrouped = (x: OptionsOrGroups<Option, GroupBase<Option>>): x is GroupBase<Option>[] =>
-    (x as readonly GroupBase<Option>[])[0]?.options !== undefined;
+export type Option = any;
 
 interface CommonProps extends Omit<ReactSelectProps<Option>, 'onChange'> {
-    withDropdownIndicator?: boolean;
+    hideDropdownIndicator?: boolean;
     isClean?: boolean;
     label?: ReactNode;
-    wrapperProps?: Record<string, any>;
     size?: InputSize;
     noError?: boolean;
     bottomText?: ReactNode;
```

Comment:
Could this be a `string` instead? Otherwise, storybook `argTypes` need:

```ts
bottomText: {
    control: 'text',
}
```

Reply:
Our `Translation` renders a `span`, so it's `ReactNode` here. But also it allows for some text to be **bold** for example!